zebra/zebra_rib.c

```diff
@@ -1080,6 +1080,9 @@ void rib_install_kernel(struct route_node *rn, struct route_entry *re,
 
 	switch (ret) {
 	case ZEBRA_DPLANE_REQUEST_QUEUED:
+		SET_FLAG(re->status, ROUTE_ENTRY_QUEUED);
+		if (old)
+			SET_FLAG(re->status, ROUTE_ENTRY_QUEUED);
```

Comment:
guessing the last line should use `old->status` instead

Reply:
fixed